.gitmodules

```diff
@@ -1,5 +1,5 @@
 [submodule "externals/Catch"]
-	path = externals/Catch
+	path = externals/Catch2
```

Comment:
Renamed the submodule. Locally developpers that already have a repo will need:

```
/bin/rm -Rf externals/Catch*
git submodule update --init --recursive --force
```

Reply:
@ibell FYI

Reply:
Thanks!